mods/cnc/audio/music.yaml

```diff
@@ -1,103 +1,63 @@
-aoi: Act On Instinct
-	VolumeModifier: 0.65
+# Base game
 airstrik: Air Strike
 	VolumeModifier: 0.9
-ccthang: C&C Thang
+j1: Untamed Land (Jungle)
 	VolumeModifier: 0.7
-ind2: Canyon Chase (Industrial 2)
+jdi_v2: Just Do It Up 2 (Take 'Em Out)
+	VolumeModifier: 0.65
+radio: Radio
 	VolumeModifier: 0.7
-heavyg: Demolition (Heavy Gear)
```

Comment:
You forgot to re-add this, should be the first track!

Reply:
He added it into CovOps. Still wrong though.